### PR TITLE
Convert datetime fields to datetime type at model level.

### PR DESCRIPTION
**Pull request by ekoval**
Convert datetime fields to datetime type at model level.

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: teamsupport/models.py [hunk 1]
```diff
@@ -7,9 +9,19 @@
 
 
 class XmlModel(object):
+    datetime_fields = ('DateCreated', 'DateModified')
+    datetime_format = '%m/%d/%Y %H:%M %p'
```
Comment (RayeN) [inline teamsupport/models.py]: I think you've missed to use it.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: teamsupport/models.py [hunk 2]
```diff
@@ -7,10 +8,19 @@
 
 
 class XmlModel(object):
+    datetime_fields = ('DateCreated', 'DateModified')
+
     def __getattr__(self, name):
-        if self.data.find(name) is not None:
-            return self.data.find(name).text
-        raise AttributeError(name)
+        value = None
```
Comment (RayeN) [inline teamsupport/models.py]: this is unnecessary

File: requirements.txt [hunk 3]
```diff
@@ -5,5 +5,6 @@ funcsigs<0.5
 nose<2.0.0
 pinocchio<1.0.0
 property-caching>=1.0.3
+python-dateutil==2.4.2
```
Comment (RayeN) [inline requirements.txt:8]: Don't forget to add it to `setup.py`

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: setup.py [hunk 4]
```diff
@@ -24,6 +24,7 @@
     'property-caching>=1.0.3',
     'querylist>=0.2.0',
     'six==1.9.0',
+    'python-dateutil==2.4.2',
```
Comment (RayeN) [inline setup.py]: Probably better to use `python-dateutil<3.0.0`

Comment (RayeN) [inline setup.py]: Also noticed 'mock>=1.2.0', in test_requirements - it should also be fixed to be inline with requirements.txt

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: setup.py [hunk 5]
```diff
@@ -23,7 +23,9 @@
     'lxml>=3.4.4',
     'property-caching>=1.0.3',
     'querylist>=0.2.0',
-    'six==1.9.0',
+    'six<2.0.0',
+    'python-dateutil<3.0.0',
+    'mock<1.2.0',
```
Comment (RayeN) [inline setup.py]: mock shouldn't be here.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by RayeN** [comment]
:+1:

**Pull request merged**